### PR TITLE
Cleanup lambda

### PR DESCRIPTION
**Pull request by jackmurdoch**
https://github.com/covidgreen/covid-green-lambdas/pull/19

### CODE REVIEW

**Comment by pmcgrath** [comment]
@jackmurdoch 
Will wait on https://github.com/nearform/covid-green-infra/pull/26
When that is merged I will rebase and update this PR branch with the new custom runtime config - so all are the same

File: lambda-cleanup.tf [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    aws_ssm_parameter.db_reader_host.arn,
+    aws_ssm_parameter.db_ssl.arn,
+    aws_ssm_parameter.security_code_lifetime_mins.arn,
+    aws_ssm_parameter.upload_token_lifetime_mins.arn
+  ]
+  aws_secret_arns                = [data.aws_secretsmanager_secret_version.rds_read_write.arn]
+  cloudwatch_schedule_expression = var.cleanup_schedule
+  config_var_prefix              = local.config_var_prefix
+  handler                        = "cleanup.handler"
+  kms_writer_arns                = [aws_kms_key.sqs.arn]
```
Comment (pmcgrath) [inline lambda-cleanup.tf]: Should this be kms_reader_arns ? or do we even need - do we do something with SQS in this lamda ?

File: lambda-cleanup.tf [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  enable = true
+  name   = format("%s-cleanup", module.labels.id)
+
+  aws_parameter_arns = [
+    aws_ssm_parameter.db_database.arn,
+    aws_ssm_parameter.db_host.arn,
+    aws_ssm_parameter.db_port.arn,
+    aws_ssm_parameter.db_reader_host.arn,
+    aws_ssm_parameter.db_ssl.arn,
+    aws_ssm_parameter.security_code_lifetime_mins.arn,
```
Comment (pmcgrath) [inline lambda-cleanup.tf:19]: Do we need these ?
- security_code_lifetime_mins
- upload_token_lifetime_mins

May well do, just want to confirm

File: lambda-cleanup.tf (hunk 1 above)
Comment (jackmurdoch) [reply lambda-cleanup.tf]: No, these are not required - didn't realise they were for SQS

File: lambda-cleanup.tf (hunk 2 above)
Comment (jackmurdoch) [reply lambda-cleanup.tf:19]: Yes, these two are used in the lambda code

File: lambda-cleanup.tf [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Trigger:
+#	Cloudwatch cron schedule
+# Resources:
+#	KMS
```
Comment (pmcgrath) [inline lambda-cleanup.tf]: Will remove this based on comment below

File: lambda-cleanup.tf (hunk 1 above)
Comment (pmcgrath) [reply lambda-cleanup.tf]: Will remove

**Comment by pmcgrath** [comment]
@jackmurdoch We need 
- a short description in the README
- a CHANGELOG entry


## Questions
1) If this is a required lambda - do the NYS guys know about/require this ?
- If yes we probably need to get an upstream PR completed sooner
2) Is the source for this in an upstream open source repo ?

**Comment by jackmurdoch** [comment]
> 1. If this is a required lambda - do the NYS guys know about/require this ?
> * If yes we probably need to get an upstream PR completed sooner

Yes -required lambda. All my recent PRs have associated code changes, so it would be good to get this repo pushed upstream sooner rather than later.

> 2. Is the source for this in an upstream open source repo ?

Yes. https://github.com/covidgreen/covid-green-lambdas/pull/19

**Comment by pmcgrath** [comment]
Yep would like to get pushed upstream @dgonzalez Can we prioritize this today ? v0.1.4

**Comment by dgonzalez** [comment]
Yes. Leave it with me. I will create an upstream PR and I will get the guys in NY up to speed. THanks!

**Review by dgonzalez — APPROVED** [review]

**Pull request merged**